frontend/src/components/App.js

```diff
@@ -223,53 +223,54 @@ const App = ({userID, usersByID, logOut, get}) => {
             className="App__sidebar"
             breakpoint="md"
             collapsedWidth={80}
-             // Ant requires a width, so pick one relative to the sidebar font-size. You can use 'auto' but then
+            // Ant requires a width, so pick one relative to the sidebar font-size. You can use 'auto' but then
             // the sidebar width changes whenever a submenu is expanded or collapsed.
-            width={'17em'} 
-            style={{overflow: 'auto'}}
+            width={'17em'}
+            style={{ overflow: 'auto' }}
           >
-            <div style={{display: 'flex', alignContent: 'baseline', justifyContent: 'left', textAlign: 'center'}}>
+            <div style={{ display: 'flex', alignContent: 'baseline', justifyContent: 'left', textAlign: 'center' }}>
               <Title style={titleStyle} className="App__title">
                 <div>
                   <b>F</b><span>reeze</span><b>M</b><span>an</span>
                 </div>
               </Title>
               { // Display a spinner while the initial data is being fetched at startup 
                 !isInitialized &&
-                  <div style={{display: 'flex', flexDirection: 'column', justifyContent: 'center'}}>
-                    <Spin size="small" indicator={loadingIcon}/>
-                  </div>
+                <div style={{ display: 'flex', flexDirection: 'column', justifyContent: 'center' }}>
+                  <Spin size="small" indicator={loadingIcon} />
+                </div>
               }
             </div>
             {isLoggedIn &&
-                <div className='App__jumpBar'>
-                  <JumpBar />
-                </div>
-              }
-              
+              <div className='App__jumpBar'>
+                <JumpBar />
+              </div>
+            }
+
+            <Menu
+              theme="dark"
+              mode="inline"
+              selectedKeys={matchingMenuKeys(MENU_ITEMS)}
+              style={{ flex: 1 }}
+            >
+              {MENU_ITEMS.map(renderMenuItem)}
+            </Menu>
+            {isLoggedIn &&
               <Menu
                 theme="dark"
                 mode="inline"
                 selectedKeys={matchingMenuKeys(MENU_ITEMS)}
-                style={{flex: 1}}
+                style={{ flex: 1 }}
                 defaultOpenKeys={['definitions']} // Submenus should be open by default
```

Comment:
This Menu doesn't contain the Definitions section - that's in MENU_ITEMS, so you should move `defaultOpenKeys` to the previous Menu.

menuItems just contains the About, Profile, and Sign Out items.